From foo@baz Tue Apr  9 12:12:43 2002
To: Greg KH <greg@kroah.com>
Date: 02 Mar 2005 11:58:47 -08:00
From: greg@kroah.com
Subject: [PATCH] I2C: just delete the id field, let's not delay it any longer

Becides, sparse keeps complaining when it sees this attribute within a structure...

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


diff -Nru a/include/linux/i2c.h b/include/linux/i2c.h
--- a/include/linux/i2c.h	2005-03-02 16:33:51 -08:00
+++ b/include/linux/i2c.h	2005-03-02 16:33:51 -08:00
@@ -144,7 +144,6 @@
  * function is mainly used for lookup & other admin. functions.
  */
 struct i2c_client {
-	__attribute__ ((deprecated)) int id;
 	unsigned int flags;		/* div., see below		*/
 	unsigned int addr;		/* chip address - NOTE: 7bit 	*/
 					/* addresses are stored in the	*/
